integration/tests/cook/test_basic.py

```diff
@@ -24,15 +25,33 @@ def setUp(self):
         util.wait_for_cook(self.cook_url)
 
     def get_job(self, job_uuid):
```

Comment:
The advantage of keeping this function is that each test doesn't have to pass `self.cook_url` as the first argument. I think it's worth it.